partitura/utils/music.py

```diff
@@ -3392,6 +3279,27 @@ def tokenize(
     return tokens
 
 
+def step2pc(step, alter):
+    """
+    Convert a step to a pitch class.
```

Comment:
New proposal for Documentation "Convert a tonal pitch class (i.e., step + alter) to a pitch class (i.e., integer in [0,11])